tests/specs/node/node_fs_cp/main.mjs

```diff
@@ -0,0 +1,23 @@
+import fs from "node:fs/promises";
+import path from "node:path";
+
+const src = path.join(import.meta.dirname, "source");
+const target = path.join(import.meta.dirname, "dist");
+
+try {
+  await fs.rm(target, { recursive: true });
+} catch {
+  // ignore
+}
+
+await fs.cp(src, target, { recursive: true, force: true });
+await fs.cp(src, target, { recursive: true, force: true });
```

Comment:
FYI: We don't yet do proper handling of the options here, but these are the ones that work in Node. So this test might also be helpful once we properly parse the options argument.